module-log-parser/src/main/kotlin/org/quiltmc/community/cozy/modules/logs/parsers/LoaderParser.kt

```diff
@@ -13,7 +13,7 @@ import org.quiltmc.community.cozy.modules.logs.data.Order
 import org.quiltmc.community.cozy.modules.logs.types.LogParser
 
 private val PATTERNS = mapOf(
-	"\n\\|[\\s\\d]+\\| Quilt Loader\\s+\\| quilt_loader\\s+\\| (\\S+).+\n"
+	"\\| Quilt Loader\\s+\\| quilt_loader\\s+\\| (\\S+).+"
```

Comment:
Couldn't this be replaced with a [kotlin raw string](https://kotlinlang.org/docs/strings.html#raw-strings) to reduce the amount of escaping ?

Reply:
Probably. I don't think any other regexes use that but it would be an improvement to change them all

Reply:
but if any of them use `$` we have to use this ugliness `"""${'$'}"""`

Reply:
~~just wait until you hear about kotlin backtick escaping allowing <code>$\`$\`</code> by referencing a variable instead of the long escape~~

Reply:
Is this a blocking issue for this PR?